docs/BASIC_SETUP.md

```diff
@@ -4,22 +4,20 @@
 
 mLRS can work with any radio which provides an SBus output, which should be really every radio. This page describes this most basic setup.
 
-In this setup, the radio only feeds the rc data to the mLRS Tx module (via SBus) but there is no communication between radio and mLRS Tx module with regards to the serial/MAVLink data stream. There is thus no such thing as telemetry in the radio. The serial/MAVLink data stream is available via a UART port on the mLRS Tx module, and it is up to you how to make use of it.
+In this setup, the radio only feeds the RC data to the mLRS Tx module (via SBus) but there is no communication between radio and mLRS Tx module otherwise. There is thus no such thing as telemetry in the radio. The serial/MAVLink data stream is available via the serial/UART port on the mLRS Tx module, and it is up to you how to make use of it. Also, there is no such thing as a Lua script for configuration, configuration of the mLRS sytem has to be done via the CLI (or OLED if available).
```

Comment:
How about 'In this setup, the radio only sends RC data to the mLRS Tx module using the SBus protocol.  As SBus is a unidirectional protocol, there is no communication from the mLRS Tx module to the radio. This means that it is not possible to have telemetry on the radio. The serial/MAVLink data stream is available via the serial/UART port on the mLRS Tx module, and it is up to you how to make use of it. When using SBus, the only options for configuration are the CLI or OLED (if available).  It is not possible to use the Lua when using SBus.'

Reply:
FANTASTIC! done